src/Compilers/Core/Portable/AdditionalTextFile.cs

```diff
@@ -42,7 +42,7 @@ public AdditionalTextFile(CommandLineSourceFile sourceFile, CommonCompiler compi
         /// Returns a <see cref="SourceText"/> with the contents of this file, or <c>null</c> if
         /// there were errors reading the file.
         /// </summary>
-        public override SourceText GetText(CancellationToken cancellationToken = default(CancellationToken))
+        public override SourceText GetText(CancellationToken cancellationToken = default)
```

Comment:
Yes, I'm okay with this (and we're likely to require it for API consumption stuff that @rchande is working on anyway).  However, we should clearly update the building and testing/contributing guidelines to make that clear as part of this change.